src/app/content/hooks/receiveContent.ts

```diff
@@ -12,9 +12,24 @@ import { content as contentRoute } from '../routes';
 import * as select from '../selectors';
 import { getCanonicalUrlParams } from '../utils/canonicalUrl';
 import { createTitle, getPageDescription } from '../utils/seoUtils';
+import { attributionValues } from '../components/Attribution';
```

Comment:
i don't really like the redux code importing from the react code, can you move this logic to a utils file?